src/ArduinoIoTCloudTCP.cpp

```diff
@@ -256,8 +256,8 @@ int ArduinoIoTCloudTCP::begin(bool const enable_watchdog, String brokerAddress,
   addPropertyReal(_ota_cap, _device_property_container, "OTA_CAP", Permission::Read);
   addPropertyReal(_ota_error, _device_property_container, "OTA_ERROR", Permission::Read);
   addPropertyReal(_ota_img_sha256, _device_property_container, "OTA_SHA256", Permission::Read);
-  addPropertyReal(_ota_url, _device_property_container, "OTA_URL", Permission::ReadWrite).onSync(CLOUD_WINS);
-  addPropertyReal(_ota_req, _device_property_container, "OTA_REQ", Permission::ReadWrite).onSync(CLOUD_WINS);
+  addPropertyReal(_ota_url, _device_property_container, "OTA_URL", Permission::ReadWrite);
```

Comment:
This is removed why? Because now the updating of the values are enforced by discrete functions during the normal program flow?

Reply:
Yes you are right, this was added to support deferred ota some tima ago. But with this new flow is noy needed anymore.
To keep OTA_URL and OTA_REQ in sync with the cloud we proceed like this:
1 - The device sends to the cloud  OTA_CAP, OTA_ERROR and OTA_SHA
2 - The device subscribes to the device topic
3 - Cloud replies with THING_ID OTA_REQ and OTA_URL